resources/language/resource.language.fi_fi/strings.po

```diff
@@ -130,84 +129,84 @@ msgstr "Ominaisuudet"
 
 msgctxt "#30607"
 msgid "Number of episodes before \"Still there?\" query"
-msgstr "Jaksojen määrä ennen \"Vielä paikalla?\" ilmoitusta"
+msgstr "\"Oletko vielä paikalla?\" -ilmoitusta edeltävä jaksomäärä"
 
 msgctxt "#30609"
 msgid "Duration in seconds for notification to be shown"
-msgstr "Kesto sekunneissa, kuinka kauan ilmoitus näytetään"
+msgstr "Ilmoituksen näyttöaika sekunneissa"
 
 msgctxt "#30611"
 msgid "Customize autoplay duration by episode length"
-msgstr "Mukauta automaattisen toiston kesto jakson pituuden mukaan"
+msgstr "Suhteuta automaattisen toiston aloituskohta jaksojen pituuteen"
 
 msgctxt "#30613"
 msgid "All episodes  [COLOR gray]any length[/COLOR]"
-msgstr "Kaikki jaksot  [COLOR gray]kaikki pituudet[/COLOR]"
+msgstr "Kaikki jaksot [COLOR gray]kaikki pituudet[/COLOR]"
 
 msgctxt "#30615"
 msgid "Very short episodes  [COLOR gray]< 10 mins[/COLOR]"
-msgstr "Todella lyhyet jaksot  [COLOR gray]< 10 min[/COLOR]"
+msgstr "Erittäin lyhyet jaksot [COLOR gray]< 10 minuuttia[/COLOR]"
 
 msgctxt "#30617"
 msgid "Short episodes  [COLOR gray]> 10 mins[/COLOR]"
-msgstr "Lyhyet jaksot  [COLOR gray]> 10 min[/COLOR]"
+msgstr "Lyhyet jaksot [COLOR gray]> 10 minuuttia[/COLOR]"
 
 msgctxt "#30619"
 msgid "Medium episodes  [COLOR gray]> 20 mins[/COLOR]"
-msgstr "Keskipituiset jaksot  [COLOR gray]> 20 min[/COLOR]"
+msgstr "Keskipituiset jaksot [COLOR gray]> 20 minuuttia[/COLOR]"
 
 msgctxt "#30621"
 msgid "Long episodes  [COLOR gray]> 40 mins[/COLOR]"
-msgstr "Pitkät jaksot  [COLOR gray]> 40 min[/COLOR]"
+msgstr "Pitkät jaksot [COLOR gray]> 40 minuuttia[/COLOR]"
 
 msgctxt "#30623"
 msgid "Very long episodes  [COLOR gray]> 60 mins[/COLOR]"
-msgstr "Todella pitkät jaksot  [COLOR gray]> 60 min[/COLOR]"
+msgstr "Erittäin pitkät jaksot [COLOR gray]> 60 minuuttia[/COLOR]"
 
 msgctxt "#30700"
 msgid "Expert"
 msgstr "Asiantuntija"
 
 msgctxt "#30703"
 msgid "Disable Up Next"
-msgstr "Poista käytöstä Up Next"
+msgstr "Poista Up Next käytöstä"
 
 msgctxt "#30705"
 msgid "Log level"
-msgstr "Lokitaso"
+msgstr "Lokitietojen laajuus"
 
 msgctxt "#30800"
 msgid "Developer"
 msgstr "Kehittäjä"
 
 msgctxt "#30801"
 msgid "Test how the user interface looks"
-msgstr "Kokeile miltä käyttöliittymä näyttää"
+msgstr "Esikatsele käyttöliittymän ulkoasua"
 
 msgctxt "#30803"
 msgid "Enable DEMO mode"
-msgstr "Ota käyttöön DEMO-tila"
+msgstr "Käytä DEMO-tilaa"
 
 msgctxt "#30805"
 msgid "Show an UpNext pop-up…"
-msgstr "Näytä UpNext ponnahdusikkuna"
+msgstr "Näytä Seuraavaksi-ilmoitus..."
 
 msgctxt "#30807"
 msgid "Show an UpNextSimple pop-up…"
-msgstr "Näytä UpNextSimple ponnahdusikkuna"
+msgstr "Näytä yksikertaisempi Seuraavaksi-ilmoitus..."
 
 msgctxt "#30809"
 msgid "Show a StillWatching pop-up…"
-msgstr "Näytä StillWatching ponnahdusikkuna"
+msgstr "Näytä Katselen edelleen -ilmoitus..."
 
 msgctxt "#30811"
 msgid "Show a StillWatchingSimple pop-up…"
-msgstr "Näytä StillWatchingSimple ponnahdusikkuna"
+msgstr "Näytä Katselen edelleen -ilmoitus..."
```

Comment:
I have no understanding of Finnish but this translated string is exactly the same as `#30809`. Is it maybe missing "yksikertaisempi"?

Reply:
Indeed it should, sorry about that! Fixed now.